feature/gnmi/gnmi_subscriptionlist_test/README.md

```diff
@@ -10,8 +10,8 @@ This is to test for gNMI `Subscription` to multiple paths with different `Subscr
   * Send a single `SubscribeRequest` message to the DUT with a **SubcriptionList** and **SubscriptionMode** matching the "Telemetry Parameter Coverage" section below. Use `Stream` mode for the `SubcribeRequest`.
   * Ensure that the implementation successfully allows subscription to all the paths mentioned below and a `SubscribeResponse` message is received by the client with the `sync_reponse` field set to `true`. The RPC via which the `SubscribeRequest` was recieved should eventually be closed by the client.
 ### GNMI-2.2: Change SubscriptionModes in the subscription list and verify receipt of sync_response:
-  * In the "Telemetry Parameter coverage" section below, change the `Subscribe` message for each of the paths with `SubscriptionMode` as `ON_CHANGE` to `TARGET_DEFINED` and the ones that are `TARGET_DEFINED` to `SAMPLE` and send all the subscribe messages in a single `SubscribeRequest` message to the DUT. Confirm that a `SubscribeResponse` message is received by the client with the `sync_reponse` field set to `true`. The client should then close the RPC session
-  * Again, switch the `SubscriptionMode` in each `Subscription` message to its original state i.e. from `TARGET_DEFINED` to `SAMPLE` and from `SAMPLE` to `TARGET_DEFINED` and resend the `SubscriptionRequest` with `Mode` as `STREAM`. Confirm that thhe DUT is responding back to the client with a `SubscriptionResponse` and the `Sync_Response` field set to `true`
+  * In the "Telemetry Parameter coverage" section below, change the `Subscribe` message for each of the paths with `SubscriptionMode` as `ON_CHANGE` to `TARGET_DEFINED` and the ones that are `TARGET_DEFINED` to `SAMPLE` w/ a sampe_interval of 10secs and send all the subscribe messages in a single `SubscribeRequest` message to the DUT. Confirm that a `SubscribeResponse` message is received by the client with the `sync_reponse` field set to `true`. The client should then close the RPC session
+  * Again, switch the `SubscriptionMode` in each `Subscription` message to its original state i.e. from `TARGET_DEFINED` to `ON_CHANGE` and from `SAMPLE` to `TARGET_DEFINED` and resend the `SubscriptionRequest` with `Mode` as `STREAM`. Confirm that the DUT is responding back to the client with a `SubscriptionResponse` and the `Sync_Response` field set to `true`
```

Comment:
Do you want     * /interfaces/interface/state/counters/in-unicast-pkts to be collected in ON_CHANGE mode?  I think we don't want counters in ON_CHANGE?  counters only need to support TARGET_DEFINED and SAMPLE mode.  Right?

There is no 'original state' of SAMPLE?  The only original states below are ON_CHANGE and TARGET_DEFINED.  Maybe you could reword this to not be conditional on the previous steps.  Just say what mode you want for which group of paths.  Give the path groups a name to simplify.

Reply:
I am not sure if we are on the same page. Heres what is happening

1. Start off the testing w/ the paths in "Telemetry Parameter Coverage" section with the "SubscriptionMode" as stated.
2. Then switch the ones that are ON_CHANGE to TARGET_DEFINED and the TARGET_DEFINED ones to SAMPLE
3. Then revert the state in "2" to its Original form i.e. TARGET_DEFINED to ON_CHANGE and SAMPLE to TARGET_DEFINED
When we follow "3", then we restore "Telemetry Parameter Coverage" to how it is stated in the section below and eqvt to "1"
 In that sense then: /interfaces/interface/state/counters/in-unicast-pkts becomes TARGET_DEFINED and not ON_CHANGE.

Reply:
Ok, got it.